src/test/specs/ExpectNoAttributeValueSpec.ts

```diff
@@ -18,4 +18,14 @@ describeCommon('expectNoAttributeValue', () => {
   it('Exact match error thrown', () => {
     expect(() => BrowserUtils.expectNoAttributeValue(EMPTY_DIV, 'align', 'center')).to.throw(Error);
```

Comment:
add error message validation

Reply:
Unable to catch error message for validation.

Reply:
because it has `).to.throw(Error).to.throw(Error);` please fix

Reply:
fixed